packages/office-ui-fabric-react/src/components/Button/sass/ButtonBase.scss

```diff
@@ -46,11 +46,4 @@
   .ms-Button-description, .ms-Button-icon {
```

Comment:
Agreed, we should be extending the props as necessary for each button type. This is one benefit of creating a distinct component for each button type.